app/javascript/chat/actions/actions.js

```diff
@@ -129,25 +140,39 @@ export function getChannels(
   });
 }
 
-export function getUnopenedChannelIds(successCb) {
-  fetch('/chat_channels?state=unopened_ids', {
+export async function getUnopenedChannelIds() {
+  const response = await request('/chat_channels?state=unopened_ids', {
     credentials: 'same-origin',
-  })
-    .then((response) => response.json())
-    .then((json) => {
-      successCb(json.unopened_ids);
-    });
+  });
+
+  return response.json()
+  // fetch('/chat_channels?state=unopened_ids', {
+  //   credentials: 'same-origin',
+  // })
+  //   .then((response) => response.json())
+  //   .then((json) => {
+  //     successCb(json.unopened_ids);
+  //   });
 }
 
-export function getContent(url, successCb, failureCb) {
-  fetch(url, {
-    Accept: 'application/json',
-    'Content-Type': 'application/json',
+// export function getContent(url, successCb, failureCb) {
+//   fetch(url, {
+//     Accept: 'application/json',
+//     'Content-Type': 'application/json',
+//     credentials: 'same-origin',
+//   })
+//     .then((response) => response.json())
+//     .then(successCb)
+//     .catch(failureCb);
+// }
+
+export async function getContent(url) {
+  const response = await request(url, {
```

Comment:
`'GET'` is the default and `credentials: 'same-origin'` are included automatically when using the `request` utility function. See lines https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js#L27 and  https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js#L47